package.json

```diff
@@ -34,6 +35,7 @@
     "karma-spec-reporter": "^0.0.20",
     "pa11y": "4.0.1",
     "pa11y-reporter-json-oldnode": "1.0.0",
-    "plato": "1.2.2"
+    "plato": "1.2.2",
+    "squirejs": "^0.1.0"
```

Comment:
Reviewers: I tried upgrading to version 0.2.1, but the API is incompatible in how it deals with mocking constructors. After a few failed attempts, I decided it was simpler to stick with the version we're using. I'm not a big fan of Squire anyway, so I'd like to see us stop using it before too long. It seems like a cheat to completely mock out dependencies.